__tests__/model-utils.test.js

```diff
@@ -0,0 +1,32 @@
+const { checkForeignPrimaryKey } = require("../models/model-utils");
+const db = require("../db/connection");
+
+afterAll(() => db.end());
+
```

Comment:
I would suggest that you should also reseed the data for each test - even though your util is only selecting, the db could still be effected by tests in your app.test.js :)

Reply:
I did contemplate re-seeding but I wondered if that would be unnecessary boiler plate (as you noted, it's only SELECTING). Good point though someone could manually change the data in psql as well and the test would fail when it should pass.